2021-01-14 20:36:30,135 INFO    MainThread:737818 [wandb_setup.py:_flush():70] setting env: {}
2021-01-14 20:36:30,135 INFO    MainThread:737818 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-14 20:36:30,135 INFO    MainThread:737818 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-14 20:36:30,135 INFO    MainThread:737818 [wandb_init.py:_log_setup():318] Logging user logs to /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210114_203630-2wz9en9h/logs/debug.log
2021-01-14 20:36:30,135 INFO    MainThread:737818 [wandb_init.py:_log_setup():319] Logging internal logs to /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210114_203630-2wz9en9h/logs/debug-internal.log
2021-01-14 20:36:31,928 INFO    MainThread:737818 [wandb_run.py:_console_start():1308] atexit reg
2021-01-14 20:36:31,928 INFO    MainThread:737818 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-01-14 20:36:31,928 INFO    MainThread:737818 [wandb_run.py:_redirect():1178] Redirecting console.
2021-01-14 20:36:31,928 INFO    MainThread:737818 [redirect.py:install():205] install start
2021-01-14 20:36:31,928 INFO    MainThread:737818 [redirect.py:install():220] install stop
2021-01-14 20:36:31,928 INFO    MainThread:737818 [redirect.py:install():205] install start
2021-01-14 20:36:31,928 INFO    MainThread:737818 [redirect.py:install():220] install stop
2021-01-14 20:36:31,929 INFO    MainThread:737818 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-14 20:36:45,986 INFO    MainThread:737818 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-01-14 20:36:45,986 INFO    MainThread:737818 [wandb_run.py:_restore():1250] restore
2021-01-14 20:36:45,986 INFO    MainThread:737818 [redirect.py:uninstall():224] uninstall start
2021-01-14 20:36:45,986 INFO    MainThread:737818 [redirect.py:_stop():279] _stop: stdout
2021-01-14 20:36:45,987 INFO    MainThread:737818 [redirect.py:_stop():285] _stop closed: stdout
2021-01-14 20:36:45,987 INFO    stdout    :737818 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-01-14 20:36:45,987 INFO    stdout    :737818 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-01-14 20:36:45,987 INFO    MainThread:737818 [redirect.py:_stop():291] _stop joined: stdout
2021-01-14 20:36:45,987 INFO    MainThread:737818 [redirect.py:_stop():293] _stop rd closed: stdout
2021-01-14 20:36:45,987 INFO    MainThread:737818 [redirect.py:uninstall():228] uninstall done
2021-01-14 20:36:45,987 INFO    MainThread:737818 [redirect.py:uninstall():224] uninstall start
2021-01-14 20:36:45,987 INFO    MainThread:737818 [redirect.py:_stop():279] _stop: stderr
2021-01-14 20:36:45,987 INFO    MainThread:737818 [redirect.py:_stop():285] _stop closed: stderr
2021-01-14 20:36:45,987 INFO    stderr    :737818 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-01-14 20:36:45,987 INFO    stderr    :737818 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-01-14 20:36:45,987 INFO    MainThread:737818 [redirect.py:_stop():291] _stop joined: stderr
2021-01-14 20:36:45,988 INFO    MainThread:737818 [redirect.py:_stop():293] _stop rd closed: stderr
2021-01-14 20:36:45,988 INFO    MainThread:737818 [redirect.py:uninstall():228] uninstall done
2021-01-14 20:36:46,445 INFO    MainThread:737818 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "7c90f1cb33ba407386002bedcdaf98f1"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 1087
      total_bytes: 1087
    }
  }
}

2021-01-14 20:36:48,451 INFO    MainThread:737818 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "6cd5e2131eb840b88f436e07a4012c57"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 3714
      total_bytes: 3714
    }
  }
}

2021-01-14 20:36:49,495 INFO    MainThread:737818 [wandb_run.py:_show_files():1614] logging synced files
2021-01-14 20:36:49,496 INFO    MainThread:737818 [internal.py:handle_exit():139] Internal process exited
